jbmc/regression/jbmc/parameter-annotation-not-null/test_array.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+Main.class
+--function Main.test2 --classpath `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+\[not-null-annotation-check\.1\] line \d+ Not null annotation check: FAILURE
+\[not-null-annotation-check\.2\] line \d+ Not null annotation check: SUCCESS
```

Comment:
:question: Why so few assert checks? Surely for this there should be 3 (for x, z, s) of which x and z can both fail, but y must succeed?